project_task_analytic_propagation/migrations/18.0.3.0.0/pre-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL-3.0)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.column_exists(env.cr, "project_task", "analytic_account_id"):
+        openupgrade.rename_columns(
+            env.cr,
+            {
+                "project_task": [
+                    ("analytic_account_id", "analytic_account_id_old"),
```

Comment:
Why not just renaming it to its future value?
```suggestion
                    ("analytic_account_id", "account_id"),
```

The issue is that if we do not do so the update is going to try to compute the values before the post-mig...